src/components/Header-Termsofuse/HeaderTermsofuse.jsx

```diff
@@ -1,40 +1,41 @@
 import React from 'react';
-import termsofuseLogo from '../../images/termsofuse-logo.png';
-import tan from '../../images/tan.png';
+import termsofuseLogo from '../../images/logo-for-terms-of-use.png';
 import favorite from '../../images/input-icon-first.svg';
 import arrow from '../../images/input-second-icon.svg';
+import tan from '../../images/tan-for-terms-of-use.png';
 
 const HeaderTermsofuse = () => {
   return (
     <div className='App-terms-of-use'>
       <div className='main-terms'>
-        <div className='container'>
-          <input type='checkbox' name='' id='' className='check' />
-          <ul className='menu-items'>
-            <li>
-              <p className='text'>Početna</p>
-            </li>
-            <li>
-              <p>Kategorije</p>
-            </li>
-            <li>
-              <p>Ponude</p>
-            </li>
-            <li>
-              <p>Pomoć</p>
-            </li>
-            <li>
-              <p>Kontakt</p>
-            </li>
-          </ul>
-          <div className='ham-menu'>
-            <span className='line line1'></span>
-            <span className='line line2'></span>
-            <span className='line line3'></span>
+        <div className='div-for-logo-and-text'>
+          <div className='container'>
+            <input type='checkbox' name='' id='' className='check' />
+            <ul className='menu-items'>
+              <li>
+                <p className='text'>Početna</p>
+              </li>
+              <li>
+                <p>Kategorije</p>
+              </li>
+              <li>
+                <p>Ponude</p>
+              </li>
+              <li>
+                <p>Pomoć</p>
+              </li>
+              <li>
+                <p>Kontakt</p>
+              </li>
+            </ul>
+            <div className='ham-menu'>
+              <span className='line line1'></span>
```

Comment:
This three lines can be replaced with icon from design

Reply:
Not working properly, please use custom component.

Reply:
Create my custom component